.github/workflows/ansible-test-sanity.yml

```
@@ -0,0 +1,21 @@
+---
+name: Sanity
+on:
+  pull_request:
+    paths:
+    - 'plugins/**'
+    - '.github/workflows/ansible-test-sanity.yml'
+  push:
+    paths:
+    - 'plugins/**'
+    - '.github/workflows/ansible-test-sanity.yml'
+  schedule:
+  # Every day at 3:14
+  - cron:  '14 3 * * *'
+  workflow_dispatch:
+
+jobs:
+  sanity:
+    uses: data-bene/ansible-collection-gh-workflow/.github/workflows/workflow-sanity.yml@devel
+    with:
+      fail-fast: false
```

```suggestion
      fail-fast: true
```
There's usually no reason not to do fail fast.